netmanager/src/views/pages/Map/paints.js

```diff
@@ -32,19 +32,50 @@ export const heatMapPaint = {
     22,
     57344,
   ],
-  // Color circle by earthquake magnitude
   "circle-color": [
     "interpolate",
     ["linear"],
     ["get", "predicted_value"],
     0,
     "rgba(236,222,239,0)",
     0.001,
-    "#44e527",
+    "#28a013",
     10,
```

Comment:
are these the colors that are aimed to represent the AQI on the heatmap? Thanks @FaithDaka , this is good progress.